boa/src/builtins/object/mod.rs

```diff
@@ -410,6 +410,24 @@ pub fn make_object(_: &Value, args: &[Value], ctx: &mut Interpreter) -> ResultVa
     Ok(object)
 }
 
+/// Creates a new object from the provided prototype
+///
+/// https://tc39.es/ecma262/#sec-object.create
+/// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create
+pub fn create_builtin(_: &Value, args: &[Value], interpreter: &mut Interpreter) -> ResultValue {
+    let __proto__ = args.get(0).cloned().unwrap_or_else(Value::undefined);
+    match __proto__ {
+        Value::Object(_) | Value::Null => Ok(Value::new_object_from_prototype(
+            __proto__,
+            ObjectData::Ordinary,
+        )),
+        _ => interpreter.throw_type_error(format!(
+            "Object prototype may only be an Object or null: {}",
+            __proto__
+        )),
+    }
+}
```

Comment:
The `Object.create()` method can also take arguments ([properties](https://tc39.es/ecma262/#sec-object.create)) that are defined on the created object.

Reply:
Very good point. I think this requires the functionality for the `Object.defineProperties()` which is currently not implemented, so I think this should be implemented first or at the same time.

Reply:
> Very good point. I think this requires the functionality for the `Object.defineProperties()` which is currently not implemented, so I think this should be implemented first. Would it be acceptable that I put a placeholder and open a new issue to make `Object.defineProperties()` and mention in that issue that after it is done the `Object.create()` should be improved with this functionality and some test-cases?

Yes that's fine, maybe we should have a case for when there are more arguments than one with `unimplemented!("propertiesObject is not implemented")`